src/BlogBundle/Resources/views/Default/index.html.twig

```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
```

Comment:
careful, we are working with the 3-lvl templating, you can't extends base from a simple view, but i believe this is a placeholder, can you add it in comment ?